package.json

```diff
@@ -84,7 +84,7 @@
     "@types/retry": "^0.12.0",
     "@types/semver": "^7.3.4",
     "@types/ssri": "^7.1.0",
-    "@vanilla-extract/css": "^1.1.0",
+    "@vanilla-extract/css": "^1.10.0",
```

Comment:
This was already being resolved as 1.10.0 in yarn.lock. This is actually the minimum version required so I took the opportunity to tidy this up.